capabilities/capability-service.yaml

```diff
@@ -0,0 +1,460 @@
+openapi: "3.0.1"
+info:
+  title: "SCALE Capability Service"
+  version: v0.0.3
```

Comment:
Minor point - but maybe should be v0.0.1 for first iteration

Reply:
Fixed